### DISCUSSION TITLE
Do I misunderstand the question here?

### DISCUSSION THREAD

**Discussion by hannibalmads** [TabularEditor/TabularEditor3#1224] [Learn]
I am going through the course and it is great.
I failed this question though and it puzzles me. Since one of the answers is "not possible", then **All of the above** should in my book also be "not possible".
![image](https://github.com/TabularEditor/TabularEditor3/assets/116729399/4e5abcc9-2da5-4ca4-b048-e2957bbc7d4c)

**Comment by mlonsk** [comment]
Hi @hannibalmads 
Thanks for the feedback and I quite agree that the phrasing is not quite right on this question. We will look into it.
